services/pepy/pepy-downloads.service.js

```diff
@@ -0,0 +1,53 @@
+import Joi from 'joi'
+import { nonNegativeInteger } from '../validators.js'
+import { BaseJsonService, pathParams } from '../index.js'
+import { renderDownloadsBadge } from '../downloads.js'
+
+const schema = Joi.object({
+  total_downloads: nonNegativeInteger,
+}).required()
+
+const description = `
+Python package total downloads from [Pepy](https://www.pepy.tech/).
+
+Download stats from pepy count package downloads from PyPI and known mirrors.`
```

Comment:
As noted in https://github.com/badges/shields/issues/4319#issuecomment-1697996155 I've decided to add this as "python package total downloads from pepy", not "PyPI total downloads" due to inclusion of downloads from known mirrors in these stats.